src/test/java/org/painandsuffer/items/weapons/DaggerTest.java

```diff
@@ -0,0 +1,26 @@
+package org.painandsuffer.items.weapons;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DaggerTest {
+
```

Comment:
Applicable to all Weapon test classes.
Since the weapon classes share a common structure or behavior, consider using parameterized tests or a shared test template to reduce code redundancy.